admin/cluster_containers_setup.py

```diff
@@ -0,0 +1,369 @@
+    """
+    message_action = message.get('action')
+
+    if message_action is not None:
+        msg = "%s\n" % message_action
+        sys.stdout.write(msg)
+        sys.stdout.flush()
+
+
+class ContainerOptions(usage.Options):
+    """
+    Parses the options passed as an argument to the create container script.
+    """
+    description = "Set up containers in a Flocker cluster."
+
+    optParameters = [
+        ['apps-per-node', None, 0, 'Number of application containers per node',
+         int],
+        ['app-template', None, None,
+         'Configuration to use for each application container'],
+        ['control-node', None, None,
+         'Public IP address of the control node'],
+        ['cert-directory', None, None,
+         'Location of the user and control certificates and user key'],
+        ['wait', None, None,
+         "The timeout in seconds for waiting until the operation is complete. "
+         "No waiting is done by default."],
+    ]
+
+    synopsis = ('Usage: setup-cluster-containers --app-per-node <containers'
+                'per node> --app-template <name of the file> '
+                '--control-node <IPAddress>'
+                '--cert-directory <path where all the certificates are>'
+                '[--wait <seconds to wait>]')
+
+    def postOptions(self):
+        # Mnadatory parameters
+        # Validate template
+        if self['app-template'] is not None:
+            template_file = FilePath(self['app-template'])
+            self['template'] = yaml.safe_load(template_file.getContent())
+        else:
+            raise usage.UsageError(
+                "app-template parameter must be provided"
+            )
+        # Validate app per node
+        if self['apps-per-node'] is None:
+            raise usage.UsageError("apps-per-node is a mandatory parameter")
+        else:
+            try:
+                self['apps-per-node'] = int(self['apps-per-node'])
+            except ValueError:
+                raise usage.UsageError("apps-per-node has to be an integer")
+        # Validate control node
+        if self['control-node'] is None:
+            raise usage.UsageError("control-node is a mandatory parameter")
+        else:
+            try:
+                IPAddress(self['control-node'])
+            except ValueError:
+                raise usage.UsageError("control-node has to be an IP address")
+        # Validate certificate directory
+        if self['cert-directory'] is None:
+            raise usage.UsageError("'cert-directory is a mandatory parameter")
+
+        # Validate optional parameters
+        if self['wait'] is not None:
+            try:
+                self['wait'] = int(self['wait'])
+            except ValueError:
+                raise usage.UsageError("The wait timeout must be an integer.")
+
+
+def main(reactor, argv):
+    environ = os.environ
+    add_destination(eliot_output)
+    # to_file(sys.stdout)
```

Comment:
Remove this or uncomment it. 

:question: Send logs to stderr instead of stdout.